Feature/site step 2

Horomi-103 opened the pull request:
1. Добавила карту
2. Сделала прелоадер

Commit:
Добавила интерактивную карту Google

Commit:
Исправила фон нижнего меню

Commit:
Сделала прелоадер

Commit:
1. Переделала карту через js
2. Сделала прелоадер только на карту

Commit:
1. Переделала отображение карты через подключение библиотеки
2. Попыталась сделать маркер. Не отображается

Commit:
1. Сделелала запрос на гео данны
2. Попыталась сохранить их на карте и в маркере. Не работает

Commit:
/

Commit:
1. Переделала отпределение геолокации
2. Поставила маркер

Commit:
Сделала alert при отказе доступа к геоданным

Commit:
Делаю таймер. Не работает

Commit:
1. Сделал папки под скрипты и стили
2. Сделала отдельно файлы со скриптами
3. Сделала таймер

crutch12 commented on scripts/map_sc.js:
```diff
@@ -0,0 +1,29 @@
+src="https://api-maps.yandex.ru/2.1/?apikey=c84c56d2-d002-4770-97e0-0de35bff37d5&lang=ru_RU"
```
это дичь какая-то, что этот код делает?)

Horomi-103 replied:
подключает яндекс карты

crutch12 commented on map.html:
```diff
@@ -1,7 +1,42 @@
+
+    <script src="https://api-maps.yandex.ru/2.1/?apikey=c84c56d2-d002-4770-97e0-0de35bff37d5&lang=ru_RU"
+        type="text/javascript"></script>
+    <script>
```
сделала map_sc.js, но зачем-то оставила здесь этот скрипт

Horomi-103 replied:
без него не отображается карта

crutch12 replied:
конечно не отображается, ты же не подключила скрипт map_sc.js
Думаешь я хотел, чтобы ты код просто в 2 места продублировала?)

crutch12 commented on time.html:
```diff
@@ -1,10 +1,39 @@
+    <link rel="stylesheet" href="styles/st_timer.css">
+    <link rel="stylesheet" href="scripts/time_sc.js">
+
+    <script>
```
зачем оставла этот скрипт, если вынесла в time_sc.js?

crutch12 commented on time.html:
```diff
@@ -1,10 +1,39 @@
 </head>
 
-<body>
+<body onload="WW()">
```
что это?

Horomi-103 replied:
запуск функции

crutch12 commented on time.html:
```diff
@@ -1,10 +1,39 @@
-    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/st_timer.css">
+    <link rel="stylesheet" href="scripts/time_sc.js">
```
скрипты подключаются через тэг скрипт, а не линк
так же, как ты подключала библиотеку для яндекс карт

crutch12 commented on scripts/map_sc.js:
```diff
@@ -0,0 +1,29 @@
+src="https://api-maps.yandex.ru/2.1/?apikey=c84c56d2-d002-4770-97e0-0de35bff37d5&lang=ru_RU"
+        type="text/javascript"
```
и этот

Commit:
1. Сделала кнопку "Где я?" на карте
2. Не получилось изменить название этой кнопки. Пока поставила алерт

crutch12 commented on map.html:
```diff
@@ -1,7 +1,56 @@
+                myMap.geoObjects.add(result.geoObjects);
+            })
+
+            butt = new ymaps.control.Button("Где я?");
```
я имел ввиду кнопку над картой, а не внутри неё
просто элемент `<button>`

crutch12 commented on scripts/time_sc.js:
```diff
@@ -0,0 +1,21 @@
+var s = 0;
```
я же говорил про названия переменных
переименуй тут всё, особенно WW

crutch12 commented on scripts/time_sc.js:
```diff
@@ -0,0 +1,21 @@
+}
+setInterval(WW, 1000);
+WW();
+/*function STOP(){
```
если код не нужен - удали его

crutch12 commented on map.html:
```diff
@@ -1,7 +1,56 @@
+            myMap.controls.add(butt, { float: 'right' });
+
+            butt.events.add('click', function (e) {
+                geolocation.get({
```
если код дублируется: надо выносить его в функцию (не забывай, что в функции можно объявить аргументы и передать их

Commit:
1. Убрала алерт при отклонении запроса на геолокацию
2. Переделала кнопку "Где я?". При согласии на геолокацию ставит маркер
3. При не согласии на геолокацию текст кнопки меняется: "не получилось... повторить?"
4. При нажатии на кнопку ещё раз и согласии на геолокацию, текст меняется на "где я?"

Commit:
Переделала таймер

Commit:
1. Изменила названия классов
2. Переделала кнопку <button>
3. Переделала ей условия при нажатии

Commit:
1. Поправила прелоадер
2. убрала поворяющийся код

Commit:
1. Подняли сервер
2. Сделала переход по страницам

crutch12 committed:
feat: переинициализация скриптов из #content

Commit:
1. Добавила скрипт таймера на все страницы в head
2. Сделала изменение title при переходе по страницам

Commit:
Перенесла библиотеку карт в content

Commit:
Почистила от ненужного

Commit:
Поменяла способ изменения title

Commit:
Убрала скрипты карты с index и time

crutch12 committed:
feat: поддержка кнопки "назад", поддержка нормального href, автоподставка href в goToPage

crutch12 committed:
fix: title в pushState

Commit:
Сделала выделение меню
